src/commands/registration/stor.js

```diff
@@ -39,10 +40,11 @@ module.exports = {
       });
 
       this.restByteCount = 0;
+      const filePath = path.join(this.fs.root, this.fs.cwd, fileName)
```

Comment:
If you only want the server's file path, this can be accessed through the `stream` with `stream.path`.